hadoop-ozone/interface-client/src/main/proto/OmClientProtocol.proto

```diff
@@ -1981,6 +1983,13 @@ message SnapshotMoveDeletedKeysRequest {
   repeated string deletedDirsToMove = 5;
 }
 
+message SnapshotMoveTableKeysRequest {
+  optional hadoop.hdds.UUID fromSnapshotID = 1;
```

Comment:
why is this optional ?

Reply:
From proto-3 onwards, required is deprecated, and suggested not to use: https://protobuf.dev/programming-guides/dos-donts/#add-required